howto/functional.po

```diff
@@ -361,30 +533,48 @@ msgid ""
 "need to do something different with the same stream, you'll have to create a "
 "new iterator."
 msgstr ""
+"Notez qu'il n'est possible de parcourir un itérateur que vers l'avant et "
+"qu'il est impossible de récupérer l'élément précédent, de réinitialiser "
+"l'itérateur ou d'en créer une copie. Des objets itérateurs peuvent offrir "
+"ces possibilités de façon facultative, mais le protocole d'itération ne "
+"spécifique que la méthode :meth:`~iterator.__next__`. Certaines fonctions "
+"peuvent ainsi consommer l'entièreté de la sortie d'un itérateur et, si vous "
+"devez utiliser le même flux pour autre chose, vous devrez en créer un "
+"nouveau."
 
 #: ../Doc/howto/functional.rst:265
 msgid "Data Types That Support Iterators"
-msgstr ""
+msgstr "Types de données gérant les itérateurs"
```

Comment:
J'aime pas trop la version anglaise. L'intention n'est-elle pas de dire « Les types itérables » tout simplement.

Reply:
```suggestion
msgstr "Types de données itérables"
```

Reply:
Parfait